vllm/model_executor/model_loader/tensorizer.py

```diff
@@ -338,10 +338,14 @@ def deserialize(self):
         per_second = convert_bytes(deserializer.total_tensor_bytes / duration)
         after_mem = get_mem_usage()
         deserializer.close()
+
+        logger_data = {"total_bytes_str": total_bytes_str}
```

Comment:
Maybe it's a premature optimization, but allocating multiple `logger_dicts` here seems excessive. I'd suggest reusing a single dict instance to reduce the number of object allocations in an effort to minimize the time spent in GC. Also, it seems like duration may be an important bit of data here.

I've included some numeric versions of some of the data since numbers often have different behavior than strings. E.g. they can be graphed more easily.

```suggestion
        logger_data = {
            "bytes_per_second": deserializer.total_tensor_bytes / duration,
            "bytes_per_second_str": per_second,
            "duration": duration,
            "event": "tensor-deserialize-completed",
            "total_bytes": deserializer.total_tensor_bytes,
            "total_bytes_str": total_bytes_str,
        }
```

Reply:
Actually, now that I look at it more, I'd just include all the logger data in the first log message and don't include any `logger_data` for the other logger calls. Then there's only one `logger_data` instance and all the relevant data is in one place.

Reply:
This seems like an instance where `logger_data` is likely useful.